docs/get-started.md

```diff
@@ -35,6 +35,18 @@ You can also use `phpinsights` via Docker:
 docker run -it --rm -v $(pwd):/app nunomaduro/phpinsights
 ```
 
+## Analyse a sub-directory or a specific file
+
+You can ask `phpinsights` to analyse only a directory or even a specific file by providing path with `analyse` command:
+
+```bash
+# For a directory
+./vendor/bin/phpinsights analyse path/to/analyse
```

Comment:
I don't know how this is handle by laravel artisan.

```
php artisan insights path/to/analyse 
```
or 
```
php artisan insights analyse path/to/analyse
```

Reply:
`php artisan insights path/to/analyse `

Reply:
Thanks :+1: I will update the docs with this